tests/integration/fixtures/ember-3.16/test-app/tests/unit/qunit-xstate-test.ts

```diff
@@ -0,0 +1,33 @@
+import { module, test } from 'qunit';
```

Comment:
@sukima this is there I def need your help.

can you fill out a basic state machine test here? (replacing undefined with the test model?)

Reply:
Sure